app/elements/io-top-secret.html

```diff
@@ -0,0 +1,113 @@
+<!--
+Copyright 2016 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer.html">
+<link rel="import" href="../bower_components/iron-ajax/iron-ajax.html">
+<link rel="import" href="../bower_components/iron-a11y-keys/iron-a11y-keys.html">
+<link rel="import" href="UtilBehavior.html">
+
+<dom-module id="io-top-secret">
+<link href="https://fonts.googleapis.com/css?family=Share+Tech+Mono" rel="stylesheet" type="text/css">
+<link rel="import" type="css" href="io-top-secret.css">
+<template>
+  <style>
+    :host {
+      z-index: 99999;
+    }
+
+    #ee-message-container {
+      @apply(--layout);
+      @apply(--layout-horizontal);
+    }
+  </style>
+  <iron-a11y-keys id="keys" target="[[target]]" keys="topSecret"
+                    on-keys-pressed="unlockTheSecret"></iron-a11y-keys>
+  <iron-ajax id="topSecretData"
+        url="/io2016/api/v1/topsecret"
+        handle-as="json"
+        on-response="onAjaxResponse"></iron-ajax>
+  <div id="ee-message-container" class="emc">
+    <div id="barfoo" class="barfoo" title="top-secret.json?">
+      <div class="square"></div>
+      <div id="ee-message" class="message">
+        You are speaking our language. Up for a challenge?
+        <a id="foo-bar-link" href="//bar.foo" class="lets-play">I want to play</a>
+        <a id="foo-bar-no-link" href="#" class="go-home" on-click="closeTopSecret">No thanks</a></div>
+    </div>
+  </div>
+</template>
+<script>
+  Polymer({
+    is: 'io-top-secret',
+
+    behaviors: [IOBehaviors.UtilBehavior],
+
+    properties: {
+      target: {
+        type: Object,
+        value: function() {
+          return document.body;
+        }
+      },
+      active: false
+    },
+
+    ready: function() {
+      this.$.topSecretData.generateRequest();
+    },
+    onAjaxResponse: function() {
+      this.setKeyListener();
+    },
+    setKeyListener: function() {
+      var response = this.$.topSecretData.lastResponse;
+      var keys = "";
+
+      for (var i = 0; i < response.length; i++) {
+        if (keys != "") { keys += "+"; }
+        keys += String.fromCharCode(parseInt(response[i], 2));
+      }
+
+      this.$.keys.keys = keys;
+    },
+    unlockTheSecret: function() {
+      if (!this.active) {
+        // add classes to container for the animation sequence
+        var el = document.getElementById('ee-message-container');
```

Comment:
you need to query your own shadow dom, not the document.

`var el = this.$['ee-message-container']`